handlers/tokenutil_test.go

```diff
@@ -0,0 +1,34 @@
+package handlers_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/atmiguel/cerealnotes/handlers"
+	"github.com/atmiguel/cerealnotes/models"
+	"github.com/atmiguel/cerealnotes/test_util"
+)
+
+func TestToken(t *testing.T) {
+	env := &handlers.Environment{nil, []byte("TheWorld")}
+
+	var num models.UserId = 32
+	bob, err := handlers.CreateTokenAsString(env, num, 1)
+	if err != nil {
+		panic(err)
+	}
+
+	token, err := handlers.ParseTokenFromString(env, bob)
+	if err != nil {
+		panic(err)
+	}
+
+	if claims, ok := token.Claims.(*handlers.JwtTokenClaim); ok && token.Valid {
+		test_util.Equals(t, int64(32), int64(claims.UserId))
+
+		fmt.Printf("%v %v", claims.UserId, claims.StandardClaims.ExpiresAt)
```

Comment:
Use [t.Logf](https://golang.org/pkg/testing/#T.Logf). 

Using testing's native stuff will make it so that this only prints if this test fails. Also, if you run tests in parallel, t.Logf will insure that simultaneous tests don't have their log lines interleaved.